BackofficeBundle/Resources/public/coffee/orchestraLib.coffee

```diff
@@ -164,13 +164,7 @@ activateTinyMce = (view, textarea) ->
       textarea.each ->
         if $(this).data('required')
           $(this).attr('required', 'required')
-      textarea.css(
-         display: 'block'
-         position: 'absolute'
-         width: '1px'
-         height: '1px'
-         top: '0px'
-         zIndex: '1')
+      textarea.css({display: 'block'}).addClass('focusable')
```

Comment:
Yes but can't you do this instead ? 

    textarea.addClass('displayBlock')

and in a stylesheet

    .displayBlock {display: block;}

This way there's no more css in our js